otx/algorithms/classification/adapters/mmcls/configurer.py

```diff
@@ -492,8 +492,6 @@ def configure_in_channel(cfg):
         if layer.__class__.__name__ in TRANSFORMER_BACKBONES and isinstance(output, (tuple, list)):
             # mmcls.VisionTransformer outputs Tuple[List[...]] and the last index of List is the final logit.
             _, output = output
-            if cfg.model.head.type != "VisionTransformerClsHead":
```

Comment:
This line is removed to support `CustomMultiLabelLinearClsHead` and `CustomHierarchicalLinearClshead`, etc which can be used for ViTs.